x-pack/plugins/apm/public/components/routing/templates/apm_main_template.tsx

```diff
@@ -29,14 +32,34 @@ export function ApmMainTemplate({
   pageTitle?: React.ReactNode;
   pageHeader?: EuiPageHeaderProps;
   children: React.ReactNode;
-} & EuiPageTemplateProps) {
+} & KibanaPageTemplateProps) {
   const { services } = useKibana<ApmPluginStartDeps>();
 
   const ObservabilityPageTemplate =
     services.observability.navigation.PageTemplate;
 
+  // TODO: NEEDS A DATA GET A CHECK
+  const hasData = false;
+  const noDataConfig: KibanaPageTemplateProps['noDataConfig'] = {
+    solution: 'Observability',
+    pageTitle: 'Set up APM for Observability!',
+    actions: {
+      elasticAgent: {
+        href: '/integrations/detail/apm-0.3.0/overview',
+        recommended: false,
+      },
+      beats: {
+        href: `/app/home#/tutorial/apm`,
+        recommended: true,
+      },
+    },
+    docsLink: '#',
+  };
```

Comment:
This is where we display the existing no-data screen: 
https://github.com/elastic/kibana/blob/957ef4ca9508d8967c7612120c21fcb06e040309/x-pack/plugins/apm/public/components/app/service_inventory/index.tsx#L199-L202